README.md

```diff
@@ -1,2 +1,104 @@
-# harry-potter-quarkus
-Showcase in a web application Quarkus and Infinispan
+# Quarkus demo: Infinispan Client
+
+This example showcases how to use Infinispan client with Quarkus.
+* Create stores dynamically
+* Simple get/put operations
+* Full-text query
+* Continuous query
+
+Uses Quarkus features, such as
+* REST web-service
+* Web Socket
+* Task Scheduling
+* Application properties
+* Application lifecycle events listeners
+
+# Before running the demo
+
+Quarkus uses [Infinispan Remote Client](http://infinispan.org/docs/dev/user_guide/user_guide.html#client_server).
+You need to have an Infinispan Server running before you run this demo.
+
+Option 1: Use Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+Option 2: Download and run the server
```

Comment:
For some reason the rendering on this is putting it on the same line instead of next according to github. We should fix this later, but won't let it hold it back any further.

Reply:
corrected this